src/link/Wasm.zig

```diff
@@ -753,6 +864,38 @@ pub fn lowerUnnamedConst(self: *Wasm, tv: TypedValue, decl_index: Module.Decl.In
     return atom.sym_index;
 }
 
+/// Returns the symbol index from the name of an intrinsic.
```

Comment:
Actually, this will be reused to resolve globals from relocatable object files linked on request I believe. Have a look at `addExternFn` in `MachO.zig` which is effectively performing the same thing (minus the implementation details).